### PR TITLE
Minor doc updates

### PR DESCRIPTION
**Pull request by jhamman**
cc: @yixinmao

### CODE REVIEW

File: rvic/parameters.py [hunk 1]
```diff
@@ -31,6 +31,16 @@
 # -------------------------------------------------------------------- #
 # Top level driver
 def parameters(config_file, numofproc=1):
+    '''
```
Comment (bartnijssen) [inline rvic/parameters.py]: I know that functionally it makes no differences, but you use `"""` for the other doc strings. May be nice to standardize (or just tell me "won't fix"

File: rvic/parameters.py [hunk 2]
```diff
@@ -79,7 +89,40 @@ def parameters(config_file, numofproc=1):
 
 
 def gen_uh_init(config_file):
-    """Initialize RVIC parameter"""
+    """Initialize RVIC parameters scrpt.
```
Comment (bartnijssen) [inline rvic/parameters.py]: `script`

File: rvic/parameters.py [hunk 3]
```diff
@@ -531,8 +610,23 @@ def gen_uh_final(outlets, dom_data, config_dict, directories):
 # -------------------------------------------------------------------- #
 # store_result helper function
 def store_result(result):
-    # This is called whenever foo_pool(i) returns a result.
-    # result_list is modified only by the main process, not the pool workers.
+    '''
```
Comment (bartnijssen) [inline rvic/parameters.py]: Same as before - may want to standardize on doc string indicator.

**Comment by jhamman** [comment]
@bartnijssen - I have standardized the character used for strings across all RVIC modules.  These changes include docstrings and all other string characters.  I prefer to use the single tick string character `'`, so that is what I have gone with.

**Comment by jhamman** [comment]
@bartnijssen  - this can be merged

**Pull request merged**